### ISSUE TITLE
Training details mismatch

### ISSUE THREAD

**Issue by hshreeshail** [hulianyuyy/Temporal-Lift-Pooling#8]
In the paper, it states that the model is trained for 80 epochs with the scheduler learning rates dropping epochs 40 and 60. However, in the config file [baseline.yaml](https://github.com/hulianyuyy/Temporal-Lift-Pooling/blob/main/configs/baseline.yaml), `num_epoch: 20` and `optimizer_args.step: [10, 15]`.
Which numbers are recommended to use?

**Comment by hulianyuyy** [comment]
The numbers in the paper is correct. I will correct that in this reop
